### PR TITLE
[v6r14] Bdii2CSAgent bad infromation caching problem fix

### LINKED ISSUE #2959: [v6r14p18] Bdii2CSAgent caching information
The bdii2csagent seems to be caching results from the ldap (or whatever) queries. We have gotten the same outdated information for some weeks now, and information about new CEs was not obtained until restarting the agent.

### PR DESCRIPTION
**Pull request by atsareg**
Should fix #2959

### CODE REVIEW

File: WorkloadManagementSystem/JobWrapper/JobWrapperTemplate.py [hunk 1]
```diff
@@ -59,7 +59,10 @@ def execute( arguments ):
   except Exception as e:
     gLogger.exception( 'JobWrapper failed the initialization phase', lException = e )
     rescheduleResult = rescheduleFailedJob( jobID, 'Job Wrapper Initialization', gJobReport )
-    job.sendJobAccounting( rescheduleResult, 'Job Wrapper Initialization' )
+    try:
+      job.sendJobAccounting( rescheduleResult, 'Job Wrapper Initialization' )
+    except Exception as e:
```
Comment (chaen) [inline WorkloadManagementSystem/JobWrapper/JobWrapperTemplate.py]: Out of curiosity, how could their be an exception here ?

Comment (atsareg) [inline WorkloadManagementSystem/JobWrapper/JobWrapperTemplate.py]: job object can be simply None if exception happens in its constructor

File: WorkloadManagementSystem/Agent/JobAgent.py [hunk 2]
```diff
@@ -465,9 +465,7 @@ def __submitJob( self, jobID, jobParams, resourceParams, optimizerParams, proxyC
       else:
         if 'Value' in submission:
           self.log.error( 'Error in DIRAC JobWrapper:', 'exit code = %s' % ( str( submission['Value'] ) ) )
-        # make sure the Job is declared Failed
-        self.__report( jobID, 'Failed', submission['Message'] )
-      return S_ERROR( '%s CE Submission Error: %s' % ( self.ceName, submission['Message'] ) )
+      return S_ERROR( '%s CE Error: %s' % ( self.ceName, submission['Message'] ) )
```
Comment (fstagni) [inline WorkloadManagementSystem/Agent/JobAgent.py]: What was the problem with declaring the job as failed?

Comment (atsareg) [inline WorkloadManagementSystem/Agent/JobAgent.py]: To not report it twice, the same report is done in the caller of __submitJob()

**Pull request merged**